bleak/backends/bluezdbus/client.py

```diff
@@ -570,13 +572,56 @@ def is_connected(self) -> bool:
             False if self._bus is None else self._properties.get("Connected", False)
         )
 
+    async def _acquire_mtu(self) -> None:
+        """Acquires the MTU for this device by calling the "AcquireWrite" or
+        "AcquireNotify" method of the first characteristic that has such a method.
+
+        This method only needs to be called once, after connecting to the device
+        but before accessing the ``mtu_size`` property.
+        """
+        # This will try to get the "best" characteristic for getting the MTU.
+        # We would rather not start notifications if we don't have to.
+        try:
+            method = "AcquireWrite"
+            char = next(
+                c
+                for c in self.services.characteristics.values()
+                if "write" in c.properties
```

Comment:
If I understand correctly, this might work only with write-without-response
"For client it only works with characteristic that has WriteAcquired property which relies on write-without-response Flag." https://github.com/Vudentz/BlueZ/blob/master/doc/gatt-api.txt

Reply:
You are right. I missed this since the device I was testing with supports both..